e2e/inference/README.md

```diff
@@ -29,32 +29,20 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
 ## Work with interactive mode
 To enable the interactive mode, the OpenVINO notebook CR needs to be created and integrated with RHODS.  
 1.	Click on the ```create Notebook``` option in this [link](https://github.com/red-hat-data-services/odh-deployer) from the web console and follow these [steps](https://github.com/openvinotoolkit/operator/blob/main/docs/notebook_in_rhods.md) to create the notebook CR.
-2.	Enable Intel Data Center GPU 
+2.	Enable Intel Data Center GPU - **Technical Preview feature**
 
-**Note:** The  Intel Data Center GPU option is not visible in the RHODS UI at this time of release. For more details, please refer to this [issue](https://github.com/opendatahub-io/odh-dashboard/issues/956). Until this issue is resolved, please follow the steps below to enable the Intel Data Center GPU.
+Create AcceleratoProfile in the ```redhat-ods-applications``` namespace 
 
-a.	Search for the OpenVINO Notebook Server from web console ```Search -> Notebook -> Jupyter-nb-<ocp-user>``` in the namespace ```rhods-notebooks```.
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/e2e/inference/accelerator_profile.yaml``` 
 
-b.	Navigate to notebook yaml and modify the yaml file according to the example shown below.
+3. Navigate to ```openvino-notebooks``` ImageStream and add the above created ```AcceleratorProfile``` key to the annotation field:
 
```

Comment:
Can we add a snapshot here and make it more clear for the audience?

Reply:
updated